### ISSUE TITLE
Site cards not keyboard focusable in site page 

### ISSUE THREAD

**Issue by ericcccsliu** [openclimatefix/pv-sites-mobile#121]
**Describe the bug**
In the site page, the cards (which are `Link` components) are not reachable by keyboard navigation 

**To Reproduce**

1. go to site page
2. try to keyboard focus on cards 
3. you can't :( 

**Expected behavior**
cards should be keyboard-focusable
